CHANGELOG.md

```diff
@@ -4,6 +4,28 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v5.0.0](https://github.com/voxpupuli/puppet-icingaweb2/tree/v5.0.0) (2024-08-13)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-icingaweb2/compare/v4.1.0...v5.0.0)
+
+**Breaking changes:**
+
+- Drop EL7 support [\#412](https://github.com/voxpupuli/puppet-icingaweb2/pull/412) ([lbetz](https://github.com/lbetz))
```

Comment:
do you also want to drop CentOS 8 support or do you want to keep that for now? (It's EoL as well)

Reply:
What do you think about kicking Centos completely?

Reply:
We can support whatever we want. My impression is that CentOS 9 is still widely used within my bubble. Does it cause a lot of maintenance trouble?